packager/haul.config.js

```diff
@@ -0,0 +1,83 @@
+const { createWebpackConfig } = require('haul');
+import { join, relative, resolve } from 'path';
+const webpack = require('webpack');
+
+function createHaulConfig() {
+  return {
+    platforms: {
+      ios: 'iOS',
+      android: 'Android',
+      win32: 'win32',
+      macos: 'MacOS',
+      uwp: 'UWP'
+    },
+    webpack: options => {
+      const { platform, bundle } = options;
+      const providesModuleNodeModules = [];
+
+      if (bundle) {
+        throw new Error('This config is only used to serve up a bundle file, not as a complete build solution');
+      }
+
+      //  configure webpack to produce bundle files with the same name as each entry file.
+      //  choose the appropriate bundle file extension, based on the platform.
+      const outputFilename = `index.${platform}.bundle`;
+
+      //  when using platform bundles the InitializeCore entry will be removed.  using a
+      //  dummy string here instead of looking up react-native module, so that when using
+      //  platform bundles, you don't need a direct dependency on react-native.
+      //
+      //  when not using a platform bundle, lookup the location of react-native from the
+      //  building package, so that we pickup the version that they are using.
+      //
+      const initializeCoreLocation = relative(
+        process.cwd(),
+        require.resolve('./platform-bundle-initcore-placeholder.js')
+      );
+
+      //  webpack configuration        
+      const factory = createWebpackConfig({
+        ...options,
+        entry: { index: [`./src/customfunctions.js`] },
```

Comment:
Needs to work with the bundle, and we need source maps working.

Reply:
As I mentioned, "I still need to figure out if it's possible to reference the bundle as opposed to the source files in haul.config.js and then have debugging work. I found that if I reference the bundle in haul.config.js, I can debug the bundle but not the individual .js file, which is not ideal"